lib/bots/versionbot.ts

```diff
@@ -822,7 +828,7 @@ export class VersionBot extends GithubBot.GithubBot {
     }
 
     // Has VersionBot already made commits to the branch.
-    private versionBotCommits(prInfo: GithubBotApiTypes.PullRequest): Promise<string | null> {
+    private hasVersionBotCommits(prInfo: GithubBotApiTypes.PullRequest): Promise<string | null> {
```

Comment:
Just to be pedantic, I'd expect a method named `has...` to return a boolean, rather than a string?